Tasks/AzureSpringCloudV0/package.json

```diff
@@ -27,10 +27,10 @@
     "@types/q": "1.0.7",
     "JSONPath": "^0.11.2",
     "azure-pipelines-tasks-azure-arm-rest-v2": "1.0.5",
-    "moment": "2.21.0",
+    "moment": "^2.29.4",
     "nock": "9.0.11",
     "q": "1.4.1",
-    "tar": "^6.1.0",
+    "tar": "^6.1.11",
     "uuid": "3.1.0",
     "webdeployment-common-v2": "file:../../_build/Tasks/Common/webdeployment-common-v2-2.206.0.tgz",
```

Comment:
Can we use this new package instead? - https://github.com/microsoft/azure-pipelines-tasks/tree/master/common-npm-packages/webdeployment-common-v4
Or maybe in the next PR